source/macros.tex

```diff
@@ -156,6 +156,7 @@
 
 % Code and definitions embedded in text.
 \newcommand{\tcode}[1]{\CodeStylex{#1}}
+\newcommand{\mcode}[1]{\text{\tcode{#1}}}   % \tcode in math mode
```

Comment:
@jensmaurer A tangent to nitpick the example you raised. I would write it as `$N = \tcode{last} - \tcode{first}$` instead of `$N = \tcode{last - first}$` so that the minus sign gets rendered as a minus sign instead of a code-style hyphen.

![example output](http://i.imgur.com/L3rk1JU.png)

On a more general note, I'm happy to help with any LaTeX code or questions.